### DISCUSSION TITLE
I have a tasmota device with 5 relays, matter enabled, but alexa recognize only the first device and do not use the name inserted in the matter configuration page but use the Friendly Name 1

### DISCUSSION THREAD

**Discussion by storchalberto** [arendst/Tasmota#19825] [Q&A]
### Discussed in https://github.com/arendst/Tasmota/discussions/19609

<div type='discussions-op-text'>

<sup>Originally posted by **avidNewb** September 27, 2023</sup>
Any plans to get Matter working with Smartthings? Currently, I can only add a device if it has no endpoints configured and I get a useless bridge. Any endpoint results in an error. I don't know if the issue is with Tasmota or Smartthings but is there any plan to get this supported?</div>

**Comment by s-hadinger** [comment]
Is the question about Alexa or Smartthings?
